interface/weno/newSearchRxNorm.php

```diff
@@ -0,0 +1,63 @@
+<form name="druglookup" method="post" action="">
+<input type="text" name="drug" size="15" value="<?php echo $drug; ?>">
+<input type="submit" value="Submit">
```

Comment:
translate Submit and ensure that feature works if Submit is changed to another string via translation